docs/source/appendix.rst

```diff
@@ -1,8 +1,62 @@
-Zipline API
------------
+API Reference
+-------------
```

Comment:
I think GitHub is rendering incorrectly for you?  These are all correct in source.